priv/riak_shell/riak_shell_regression1.log

```diff
@@ -80,3 +80,133 @@ family1,series1,1970-01-01T00:00:00.007Z,cloudy,27.9
+to `history` or `h`: `history 3;` or `h 3;` for example."}}.
+{{command, "help ; "}, {result, "The following functions are available
+
+Extension 'connection':
+    connect, connection_prompt, ping, reconnect, show_connection, show_cookie
+    show_nodes
+
+Extension 'debug':
+    load, observer
+
+Extension 'history':
+    clear_history, h, history, show_history
+
+Extension 'log':
+    date_log, log, logfile, regression_log, replay_log, show_log_status
+
+Extension 'shell':
+    about, q, quit, show_config, show_version
+
+You can get more help by calling help with the
+extension name and function name like 'help shell quit;'
+
+For SQL help type 'help SQL'"}}.
+{{command, "help ; "}, {result, "The following functions are available
+
+Extension 'connection':
+    connect, connection_prompt, ping, reconnect, show_connection, show_cookie
+    show_nodes
+
+Extension 'debug':
+    load, observer
+
+Extension 'history':
+    clear_history, h, history, show_history
+
+Extension 'log':
+    date_log, log, logfile, regression_log, replay_log, show_log_status
+
+Extension 'shell':
+    about, q, quit, show_config, show_version
+
+You can get more help by calling help with the
+extension name and function name like 'help shell quit;'
+
+For SQL help type 'help SQL'"}}.
+{{command, "date_log; "}, {result, "Error: invalid function call : log_EXT:date_log []
+Toggle adding a timestamp to the name of the log file with `date_log on ;`
+and off with `date_log off ;`
+The filename will be something like \"riak_shell.2016_02_15-16:42:22.log\"
+You will get a new log file for each session of riak-shell.
+
+The default can be set in the config file."}}.
+{{command, "date_log off; "}, {result, "Log files will not contain a date/time stamp."}}.
+{{command, "help; "}, {result, "The following functions are available
+
+Extension 'connection':
+    connect, connection_prompt, ping, reconnect, show_connection, show_cookie
+    show_nodes
+
+Extension 'debug':
+    load, observer
+
+Extension 'history':
+    clear_history, h, history, show_history
+
+Extension 'log':
+    date_log, log, logfile, regression_log, replay_log, show_log_status
+
+Extension 'shell':
+    about, q, quit, show_config, show_version
+
+You can get more help by calling help with the
+extension name and function name like 'help shell quit;'
+
+For SQL help type 'help SQL'"}}.
+{{command, "help sql ; "}, {result, "The following SQL help commands are supported:
+CREATE   - using CREATE TABLE statements
+DELETE   - deleting data with DELETE FROM
+DESCRIBE - examining table structures
+EXPLAIN  - understanding SELECT query execution paths
+INSERT   - inserting data with INSERT INTO statements
+SELECT   - querying data
+SHOW     - listing tables
+
+SELECT can be used with ORDER BY, GROUP BY and LIMIT clauses. It supports arithmetic on column values and has a variety of aggregation functions: COUNT, SUM, MEAN, AVG, MAX, MIN, STDDEV, STDDEV_SAMP and STDDEV_POP
```

Comment:
Add a note saying GROUP BY or any aggregation functions are not compatible with ORDER BY/LIMIT/OFFSET.